test-app/.eslintignore

```diff
@@ -0,0 +1,25 @@
+# unconventional js
+/blueprints/*/files/
+/vendor/
+
+# compiled output
+/dist/
+/tmp/
+
+# dependencies
+/bower_components/
+/node_modules/
+
+# misc
+/coverage/
+!.*
+.*/
+.eslintcache
+
+# ember-try
+/.node_modules.ember-try/
+/bower.json.ember-try
+/npm-shrinkwrap.json.ember-try
+/package.json.ember-try
+/package-lock.json.ember-try
+/yarn.lock.ember-try
```

Comment:
**suggestion**

After merging the pull requests that you've worked on, in order to update `ember-qunit` to be a v2 addon, I'd recommend that you update the configuration files such as `.eslintignore`, `.eslintrc.js`, `.npmignore`, etc., by matching the latest code from `ember-cli` blueprints.

https://github.com/ember-cli/ember-cli/tree/v4.12.1/blueprints/app/files

I think this will help us maintain the project in the future.

Reply:
Sure thing. Would like to merge and do a follow up tho

Reply:
https://github.com/emberjs/ember-qunit/issues/1092